client/my-sites/plugins/plugins-list/test/index.jsx

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import noop from 'lodash/noop';
+
+import useMockery from 'test/helpers/use-mockery';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+import { sites } from './sites-data';
+import i18n from 'lib/mixins/i18n';
+
+describe( 'PluginsList', () => {
+	let React, ReactInjection, TestUtils;
+	let PluginsList = {};
+	let siteListMock = {
+		getSelectedSite() {
+			return sites[0];
+		}
+	};
+	useFakeDom();
+	useMockery( mockery => {
+		mockery.registerSubstitute( 'matches-selector', 'component-matches-selector' );
+		mockery.registerSubstitute( 'query', 'component-query' );
+
+		let emptyComponent = require( 'test/helpers/react/empty-component' );
+		mockery.registerMock( 'my-sites/plugins/plugin-item/plugin-item', emptyComponent );
+		mockery.registerMock( 'my-sites/plugins/plugin-list-header', emptyComponent );
+		mockery.registerMock( 'analytics', { ga: { recordEvent: noop } } );
+		mockery.registerMock( 'component-classes', () => {
+			return { add: noop, toggle: noop, remove: noop }
+		} );
+		mockery.registerMock( 'lib/sites-list', () => siteListMock );
+	} );
+
+	before( () => {
+		React = require( 'react' );
+		ReactInjection = require( 'react/lib/ReactInjection' );
+		TestUtils = require( 'react-addons-test-utils' );
+
+		i18n.initialize();
+		ReactInjection.Class.injectMixin( i18n.mixin );
```

Comment:
@nb It doesn't solve the mass of mocks here but I do see how the `i18n` setup could be moved out to a common test stub. I'm not 100% sure it would work but I'm thinking I might give it a shot.